js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/RuleEvaluationOutcomes.tsx

```diff
@@ -147,6 +147,8 @@ const RightElementForEvaluations = ({
       );
     case 'WaitingOnKeysRuleEvaluationData':
       return <WaitingOnAssetKeysLink assetKeys={first.waitingOnAssetKeys || []} />;
+    case 'RequiredButNonexistentParentsRuleEvaluationData':
```

Comment:
For now display nothing on the right hand side. This looks like:
<img width="812" alt="image" src="https://github.com/dagster-io/dagster/assets/29110579/cb8ef6b4-04b4-4f2e-a413-e7ba0731592c">